test/uk/gov/hmrc/apidefinition/service/DocumentationServiceSpec.scala

```diff
@@ -79,7 +79,7 @@ class DocumentationServiceSpec extends AsyncHmrcSpec with GuiceOneAppPerSuite wi
     authorised = false
   )
 
-  val apiDefinition: ApiDefinition = ApiDefinition(
+  val apiDefinition: StoredApiDefinition = StoredApiDefinition(
```

Comment:
Variable name could be `storedApiDefinition` then don't need the type as well.

Reply:
It's fine